server/monitor_test.go

```diff
@@ -4116,13 +4116,15 @@ func TestMonitorJsz(t *testing.T) {
 			max_mem_store: 10Mb
 			max_file_store: 10Mb
 			store_dir: '%s'
+			unique_tag: az
 		}
 		cluster {
 			name: cluster_name
 			listen: 127.0.0.1:%d
 			routes: [nats-route://127.0.0.1:%d]
 		}
-		server_name: server_%d `, test.port, test.mport, tmpDir, test.cport, test.routed, test.port)))
+		server_name: server_%d
+        server_tags: [ "az:%d" ] `, test.port, test.mport, tmpDir, test.cport, test.routed, test.port, test.port)))
```

Comment:
indentation.

Reply:
Hmm. Not showing in my editor. I cloned the branch from origin to a fresh directory and also not showing.  Will research further.

Reply:
tabs vs spaces

Reply:
copy tabs from line above and paste.